pkcs11/src/new/functions/slot_token_management.rs

```diff
@@ -67,4 +70,34 @@ impl Pkcs11 {
 
         Ok(slots)
     }
+
+    pub fn init_token(&self, slot: Slot, pin: &str) -> Result<()> {
+        let pin = Secret::new(CString::new(pin)?.into_bytes());
+        // FIXME: make a good conversion to the label format
+        // 32 is the ASCII code for ' '
+        let label: [u8; 32] = [32; 32];
```

Comment:
Label will always be an empty string but I think this is fine for now

Reply:
We could just use the [existing function](https://github.com/mheese/rust-pkcs11/blob/1323d5f4cc4774235ef327d0f819804e2b6f2894/src/lib.rs#L70) but I am sure there is something simpler to do

Reply:
I'd probably go with 
```suggestion
        let label = [b` `; 32];
```